src/main/java/com/playmonumenta/networkrelay/RemotePlayerManagerPaper.java

```diff
@@ -0,0 +1,247 @@
+package com.playmonumenta.networkrelay;
```

Comment:
...I'm not checking this file thoroughly for now, I'm just going to assume it has similar issues to the Bungee implementation.